database.sql

```diff
@@ -33,6 +33,61 @@ CREATE TABLE IF NOT EXISTS `gserver` (
 	 UNIQUE INDEX `nurl` (`nurl`(190))
 ) DEFAULT COLLATE utf8mb4_general_ci COMMENT='Global servers';
 
+--
+-- TABLE user
+--
+CREATE TABLE IF NOT EXISTS `user` (
```

Comment:
Please add a comment in the file header mentioning that the order of table creation is important.

Reply:
Done